src/pages/TCY/components/Unstake/Unstake.tsx

```diff
@@ -131,9 +129,8 @@ export const UnstakeRoutes: React.FC<TCYRouteProps & { activeAccountNumber: numb
           chainId: thorchainChainId,
           accountId,
           assetId: tcyAssetId,
-          message: translate(`TCY.unstakeStatus.successSubtitle`, {
-            amount: amountCryptoPrecision,
-          }),
+          amountCryptoPrecision,
+          message: 'TCY.unstakeStatus.successSubtitle',
```

Comment:
q: wondering if `ActionGenericTransactionMetadata['message']` should be `string | [string, InterpolationOptions]` instead of string currently?